src/components/MessageLogger/Counter.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import styles from "@components/MessageLogger/Counter.module.scss";
+
+interface Props {
+  count: number;
+}
+
+export const Counter = ({ count }: Props) => {
+  return <>{count > 1 ? <label id={styles.count}>{count}</label> : null}</>;
```

Comment:
This component shouldn't have the logic of appearing and dissapearing, that should be implemented in the LineMessage